Triangulation_2/doc/Triangulation_2/CGAL/Constrained_triangulation_plus_2.h

```diff
@@ -293,6 +293,30 @@ std::size_t insert_constraints(PointIterator points_first, PointIterator points_
                                IndicesIterator indices_first, IndicesIterator indices_last);
 
 
+/*! 
+splits into constraints the graph of subconstraints.
+
+Consider the graph `g={V,E}` where `V` is the set of vertices of the
+triangulation and `E` is the set of all subconstraints of all
+constraints of the triangulation.
+
+This function splits into polylines the graph `g` at vertices of
+degree greater than 2 and at vertices for which
+`is_terminal(v)==true`.
+
+Each computed polyline is stored as a constraint of the triangulation.
+
+\warning all existing constraints will be discarded.
+
+\param is_terminal An optional function returning true if the vertex
+`v` of degree 2 is a polyline endpoint and false otherwise. If no
+function is provided, a default function is provided that returns true
+for vertices whose degree is different from 2.
```

Comment:
```suggestion
\param is_terminal An optional function returning `true` if the vertex
`v` of degree 2 is a polyline endpoint and `false` otherwise. If omitted, a function
function always returning `false` will be used, that is no degree 2 vertex will be considered as a polyline endpoint.
```

Reply:
I agree with the suggestion, but be careful that this suggestion contains a typo:
```
             a function
function
```